content/plugins/heloufir-filament-kanban.md

```diff
@@ -0,0 +1,14 @@
+---
+anystack_id: 9a833198-82cc-42db-836d-a62aaf9e0a15
+name: Filament Kanban
+slug: heloufir-filament-kanban
+author_slug: heloufir
+categories: [filament]
+description: Integrates Kanban into Filament projects, simplifying task management, progress tracking, and team collaboration, enhancing productivity and organization.
+docs_url: https://gist.github.com/heloufir/05a86302f871fa1a81ac05d693d9c9d9
```

Comment:
Please provide the docs url to the raw content of the md.

Reply:
@awcodes done you can check

Reply:
Can you please give more info? the docs url is already mentioned as the Filament docs says

Reply:
@awcodes my bad, I understand now thanks :)
I added the raw url of the docs